projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -108,7 +111,7 @@ const diffInSets = (set1: Set<any>, set2: Set<any>): any[] => {
     ]
 })
 export class IgxComboComponent extends IgxComboBaseDirective implements AfterViewInit, ControlValueAccessor, OnInit,
-    OnDestroy, EditorProvider {
+    OnDestroy, EditorProvider, AfterContentChecked {
```

Comment:
it is not implementing now `AfterContentChecked`

Reply:
Since the base is actually implementing it, I decided to leave it here as well so that it's known that the component will do something at the `AfterContentChecked` step, same goes for the pickers.